config/default.yml

```diff
@@ -3840,7 +3842,7 @@ Style/TrivialAccessors:
   # Commonly used in DSLs
   AllowDSLWriters: false
   IgnoreClassMethods: false
-  Whitelist:
+  AllowedMethod:
```

Comment:
Seems that this is missing the trailing `s`.
Should be `AllowedMethods`.

PR incoming.

Reply:
Issue: https://github.com/rubocop-hq/rubocop/issues/7530
PR: https://github.com/rubocop-hq/rubocop/pull/7529